### PR TITLE
Fixes video-game-coding-club/crypto-webpage#54

### PR DESCRIPTION
**Pull request by zmedin**
Signed-off-by: zmedin <zmedin@gmail.com>

### CODE REVIEW

File: helpers.js [hunk 1]
```diff
@@ -19,8 +19,19 @@ export function selectLanguage (language) {
...
 
 function processForm () { // eslint-disable-line no-unused-vars
-  alert('processing form (' + $('#inputtext')[0].value + ')');
-  alert('shiftLetter("a", 1) function result: ' + shiftLetter('a', 1));
+  // Get the secret key from the form.
+  var formKey = $('#key')[0].value;
+
+  // Check if formKey is a number. isNan is false if the key is
+  // empty, so let's check that too.
+  if (isNaN(formKey) || formKey === '') {
```
Comment (nicolasbock) [inline helpers.js]: Nice! Could you move this code into a separate function? You can then add some unit tests to see whether this function catches incorrect input. Thanks!

**Review by nicolasbock — COMMENTED** [review]
Nice start!

File: helpers.js (hunk 1 above)
Comment (zmedin) [reply helpers.js]: Okay, I did it.

**Review by nicolasbock — APPROVED** [review]
Thanks!

**Pull request merged**